src/functionHandlerProvider/providers/HttpServerHandlerProvider.ts

```diff
@@ -102,3 +102,118 @@ export { handler };`;
+        const handlerContent = `
+from io import BytesIO
+import traceback
+import base64
+from ${nameModule} import ${functionConfiguration.handler} as application
+
+def handler(event):
+    try:
+        # Parse HTTP information from the event
+        http_info = event.get('http', {})
+        path = event.get('path', http_info.get('path', '/'))
+
+        # Create the WSGI environment from the received HTTP event
+        environ = create_wsgi_environ(event, http_info, path)
+
+        # Response buffer
+        response_buffer = BytesIO()
+        status_code = 500  # Default status in case of error
+        headers_dict = {}
+
+        # The start_response function for handling WSGI responses
+        def start_response(status, headers, exc_info=None):
+            nonlocal status_code, headers_dict
+            status_code = int(status.split()[0])  
+            headers_dict = {key: value for key, value in headers}
+            return response_buffer.write
+
+        # Call the WSGI application and build the response
+        app_response = application(environ, start_response)
+        response_body = b''.join(app_response)
+
+        # Return the formatted response based on content type
+        return format_response(response_body, headers_dict, status_code)
+
+    except Exception as e:
+        # Log errors and return a generic 500 response
+        print("Error processing request:", traceback.format_exc())
+        return {
+            "statusCode": 500,
+            "body": "Internal Server Error",
+            "headers": {"Content-Type": "text/plain"},
+            "error": str(e)
+        }
+
+def create_wsgi_environ(event, http_info, path):
+    """
+    Creates a WSGI environment based on the HTTP information received from the event.
+    """
+    return {
+        'REQUEST_METHOD': http_info.get('method', 'GET'),
+        'PATH_INFO': path,
+        'QUERY_STRING': event.get('query', ''),
+        'REMOTE_ADDR': http_info.get('sourceIp', ''),
+        'CONTENT_TYPE': event.get('headers', {}).get('Content-Type', ''),
+        'CONTENT_LENGTH': str(len(event.get('body', ''))),
+        'wsgi.input': BytesIO(event.get('body', '').encode() if isinstance(event.get('body', ''), str) else event['body']),
+        'wsgi.errors': BytesIO(),
+        'wsgi.url_scheme': 'http',
+        # Add HTTP headers
+        **create_wsgi_headers(event.get('headers', {}))
+    }
+
+def create_wsgi_headers(headers):
+    """
+    Transforms the headers from the event into the WSGI required format (prefixing with HTTP_).
+    """
+    wsgi_headers = {}
+    for header, value in headers.items():
+        wsgi_headers[f"HTTP_{header.upper().replace('-', '_')}"] = value
+    return wsgi_headers
+
+def format_response(response_body, headers, status_code):
+    """
+    Formats the response to return, ensuring the content type is handled appropriately.
+    """
+    content_type = headers.get('Content-Type', 'text/html')
```

Comment:
What happens for `application/json` bodies?

Reply:
It will be decoded as `utf-8`

Reply:
Also, I've added an example route for this: 

`@app.route('/post-data', methods=['POST'])
def post_data():
    data = request.get_json()
    if not data:
        return {'error': 'No data found in request'}, 400
    print(f"Received data: {data}")
    return data`

https://github.com/Genez-io/flask-getting-started